vector_store_test.go

```diff
@@ -0,0 +1,339 @@
+				fmt.Fprintln(w, `{
+					id: "file-wB6RM6wHdA49HfS2DJ9fEyrH",
+					object: "vector_store.file.deleted",
+					deleted: true
+				  }`)
+			}
+		},
+	)
+
+	server.RegisterHandler(
+		"/v1/vector_stores/"+vectorStoreID+"/files",
+		func(w http.ResponseWriter, r *http.Request) {
+			if r.Method == http.MethodGet {
+				resBytes, _ := json.Marshal(openai.VectorStoreFilesList{
+					VectorStoreFiles: []openai.VectorStoreFile{
+						{
+							ID:            vectorStoreFileID,
+							Object:        "vector_store.file",
+							CreatedAt:     1234567890,
+							VectorStoreID: vectorStoreID,
+						},
+					},
+				})
+				fmt.Fprintln(w, string(resBytes))
+			} else if r.Method == http.MethodPost {
+				var request openai.VectorStoreFileRequest
+				err := json.NewDecoder(r.Body).Decode(&request)
+				checks.NoError(t, err, "Decode error")
+
+				resBytes, _ := json.Marshal(openai.VectorStoreFile{
+					ID:            request.FileID,
+					Object:        "vector_store.file",
+					CreatedAt:     1234567890,
+					VectorStoreID: vectorStoreID,
+				})
+				fmt.Fprintln(w, string(resBytes))
+			}
+		},
+	)
+
+	server.RegisterHandler(
+		"/v1/vector_stores/"+vectorStoreID+"/file_batches/"+vectorStoreFileBatchID+"/files",
+		func(w http.ResponseWriter, r *http.Request) {
+			if r.Method == http.MethodGet {
+				resBytes, _ := json.Marshal(openai.VectorStoreFilesList{
+					VectorStoreFiles: []openai.VectorStoreFile{
+						{
+							ID:            vectorStoreFileID,
+							Object:        "vector_store.file",
+							CreatedAt:     1234567890,
+							VectorStoreID: vectorStoreID,
+						},
+					},
+				})
+				fmt.Fprintln(w, string(resBytes))
+			}
+		},
+	)
+
+	server.RegisterHandler(
+		"/v1/vector_stores/"+vectorStoreID+"/file_batches/"+vectorStoreFileBatchID+"/cancel",
+		func(w http.ResponseWriter, r *http.Request) {
+			if r.Method == http.MethodPost {
+				resBytes, _ := json.Marshal(openai.VectorStoreFileBatch{
+					ID:            vectorStoreFileBatchID,
+					Object:        "vector_store.file_batch",
+					CreatedAt:     1234567890,
+					VectorStoreID: vectorStoreID,
+					Status:        "cancelling",
+					FileCounts: openai.VectorStoreFileCount{
+						InProgress: 0,
+						Completed:  1,
+						Failed:     0,
+						Cancelled:  0,
+						Total:      0,
+					},
+				})
+				fmt.Fprintln(w, string(resBytes))
+			}
+		},
+	)
+
+	server.RegisterHandler(
+		"/v1/vector_stores/"+vectorStoreID+"/file_batches/"+vectorStoreFileBatchID,
+		func(w http.ResponseWriter, r *http.Request) {
+			if r.Method == http.MethodGet {
+				resBytes, _ := json.Marshal(openai.VectorStoreFileBatch{
+					ID:            vectorStoreFileBatchID,
+					Object:        "vector_store.file_batch",
+					CreatedAt:     1234567890,
+					VectorStoreID: vectorStoreID,
+					Status:        "completed",
+					FileCounts: openai.VectorStoreFileCount{
+						InProgress: 0,
+						Completed:  1,
+						Failed:     0,
+						Cancelled:  0,
+						Total:      0,
```

Comment:
There's no need to specify 0 here and in the similar cases as 0 is the default value https://go.dev/tour/basics/12